CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## VNext
 
 ## Version 2.14.0-beta4
+- [Fix: https://github.com/microsoft/ApplicationInsights-dotnet/issues/1761](https://github.com/microsoft/ApplicationInsights-dotnet/issues/1761)
```

Comment:
Please move this to the VNext section. Beta4 has already been compiled and will be released soon.